apex/contrib/group_norm/group_norm.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Copyright (c) 2011-2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without modification, are not permit-
+# ted.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR
+# IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+# FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL NVIDIA CORPORATION BE LIABLE
+# FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+# BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+# OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+# STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+import torch
+import torch.nn.functional as F
+import torch.nn.init as init
+import group_norm_cuda
+
+from torch import Tensor
+from torch.nn.parameter import Parameter
+from torch._dynamo import disable
+from functools import partial
+
+__all__ = [
+    'GroupNormOpt',
+    'cuda_group_norm_nhwc',
+    'cuda_group_norm_nhwc_one_pass',
+    'cuda_group_norm_nhwc_two_pass',
```

Comment:
would you expect these to be used directly?
or would `GroupNormOpt` be the one-stop shop?

Reply:
I would expect user only use `GroupNormOpt`. We export two different operators here mainly due to testing requirements. `GroupNormOpt` contains a heuristic to choose one pass or two pass algorithms. During testing, we want to test every algorithms.

Reply:
I know it's a mere nit picking but how about 
- making `__all__` to include only `GroupNorm`
- importing `cuda_group_norm_nhwc_(one|two)_pass` from `apex.contrib.group_norm.group_norm` in the test
?